benchmarks/yamauchi_takei_2016_anelasticity/yamauchi_takei_2016_anelasticity.prm

```diff
@@ -0,0 +1,90 @@
+# This benchmark tests the ASPECT implementation of an initial temperature model
+# that converts absolute Vs to temperature conversion using the anelasticity model of 
+# Yamauchi & Takei 2016 (JGR). This .prm aims to recreate Figure 20 in the paper.
+# Output can be plotted and compared to results in Figure 20 by running plot_output.gmt
+# in the plot_output folder.
+
+
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set Start time                             = 0
+set End time                               = 0
+set Additional shared libraries = ./libanelasticity_temperature.so
+set Output directory                       = output_yamauchi_takei_2016_anelasticity
+
+
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X extent = 52000
+    set Y extent = 100000
+  end
+end
+
+
+subsection Initial temperature model
+  set Model name = anelastic Vs to temperature
+  set Remove temperature heterogeneity down to specified depth = 25000
+  set Use Yamauchi and Takei parameterization = true
+  set Use original density and frequency model of Yamauchi and Takei = true
+  subsection Ascii data model
+  	set Data directory       = /Users/fredrichards/aspect_master/data/initial-temperature/ascii-data/test/
```

Comment:
Can you use ``$ASPECT_SOURCE_DIR/data/`` here?

Reply:
Yes, have changed